public/index.html

```diff
@@ -43,7 +43,7 @@
       .force("center", d3.forceCenter(width / 2, height / 2));
 
   d3.json("/graphdata.json", function(error, graph) {
-    if (error) throw error;
+    if (error) return;
```

Comment:
See L95, if the client is rate limited both `graphdata.json` and `dashboarddata.json` are empty files and the `error` variable in the calls to `d3.json` is set, so no logging is *strictly* necessary.